src/Aspire.Hosting/Dashboard/DashboardLifecycleHook.cs

```diff
@@ -36,9 +45,30 @@ public Task BeforeStartAsync(DistributedApplicationModel model, CancellationToke
             AddDashboardResource(model);
         }
 
+        _dashboardLogsTask = WatchDashboardLogsAsync();
+
         return Task.CompletedTask;
     }
 
+    public async ValueTask DisposeAsync()
+    {
+        _shutdownTcs.Cancel();
+
+        if (_dashboardLogsTask is not null)
+        {
+            try
+            {
+                await _dashboardLogsTask.ConfigureAwait(false);
+            }
+            catch (Exception ex)
```

Comment:
Add catch for OperationCanceledException. Don't log for it.

Reply:
There's an exception inside with this catch.